cypress/e2e/AdminForm.cy.ts

```diff
@@ -0,0 +1,78 @@
+import { meSuperadmin } from "../intercepts/CreateAdmin";
+
+describe("create admin page", () => {
+  it("shows all elements of the form", () => {
+    meSuperadmin();
+
+    cy.visit("http://localhost:5173/admins/create");
+
+    cy.contains("Create Admin").should("be.visible");
+    cy.contains("Email address").should("be.visible");
+    cy.contains("Password").should("be.visible");
+    cy.contains("Super admin privileges").should("be.visible");
+    cy.get("input[type=checkbox]")
+      .should("be.visible")
+      .should("not.be.checked");
+    cy.contains("button", "Save")
+      .should("be.visible")
+      .should("have.attr", "type", "submit");
+    cy.contains("a", "Cancel")
+      .should("be.visible")
+      .should("have.attr", "href", "/admins");
+  });
+
+  it("creates a new admin", () => {
+    meSuperadmin();
+
+    cy.visit("http://localhost:5173/admins/create");
+
+    cy.get("input[type=email]").type("test@example.com");
+    cy.get("input[type=password]").type("test");
+    cy.get("input[type=checkbox]").click();
+    cy.contains("button", "Save").click();
+  });
+
+  it("navigates back to the admin list on cancel", () => {
+    meSuperadmin();
+
+    cy.visit("http://localhost:5173/admins/create");
+
+    cy.contains("a", "Cancel").click();
+    cy.url().should("include", "/admins");
+  });
+
+  it("shows email already taken error", () => {
+    meSuperadmin();
+
+    cy.visit("http://localhost:5173/admins/create");
+
+    cy.get("input[type=email]").type("test@example.com");
+    cy.get("input[type=password]").type("test");
+    cy.get("input[type=checkbox]").click();
+    cy.contains("button", "Save").click();
+
+  });
+
+  it("shows email is required error", () => {
+    meSuperadmin();
+
+    cy.visit("http://localhost:5173/admins/create");
+
+    cy.get("input[type=password]").type("test");
+    cy.get("input[type=checkbox]").click();
+    cy.contains("button", "Save").click();
+    cy.contains("Email address").should("be.visible");
```

Comment:
Once again, youre not testing anything here?

```suggestion
    meSuperadmin();

    cy.visit("http://localhost:5173/admins/create");

    cy.get("input[type=password]").type("test");
    cy.get("input[type=checkbox]").click();
    cy.contains("button", "Save").click();
    cy.contains("Email address").should("be.visible");
    
    cy.get("input[name=email]").then(($input) => {
      expect($input[0].validationMessage).to.eq("Please fill in this field.");
    });
```

You can verify if a form error exists like this

Reply:
https://docs.cypress.io/faq/questions/using-cypress-faq#Can-I-check-that-a-forms-HTML-form-validation-is-shown-when-an-input-is-invalid

Context here

Reply:
I tried the input:invalid but that didnt work, so stick with the get.then